src/interpreter/ByteCode.h

```diff
@@ -78,6 +79,58 @@ class FunctionType;
     F(Store32)                  \
     F(Store64)                  \
     F(FillOpcodeTable)
+#else /* !NDEBUG */
+#define FOR_EACH_BYTECODE_OP(F) \
```

Comment:
Duplicating the entire macro is not a good idea.

What about defining a F_NOP macro which is empty or `F(Nop)` depending on debug?